docs/cave/d2d-gis-shapefiles.md

```diff
@@ -36,26 +40,25 @@ Use the Apply button to save any changes you make to the default attribute setti
 
 ## Customizing the GIS Attribute Dialog
 
-You have the ability to highlight specific areas of the displayed map or map product image, and to hide other areas. These functionalities are executed through the **Attributes dialog**, commonly referred to as the "Attributes Table" because of its tabular style. To access the **Attributes dialog**, click and hold middle mouse button 3 (B3) on the Map ID in the Legend area to open the pop-up menu.
+You have the ability to highlight specific areas of the displayed map or map product image, and to hide other areas. These functionalities are executed through the **Attributes dialog**, commonly referred to as the "Attributes Table" because of its tabular style. To access the **Attributes dialog**, right click and hold mouse button 3 on the Map ID in the Legend area to open the pop-up menu.
```

Comment:
There's a lot about this "mouse button 3" which is how it's referenced in the Raytheon documentation but I don't think we use it anywhere else

Reply:
Yeah I think the middle mouse button (btn3) is only used for panning?

Reply:
well his notes were actually wrong, it's a right click hold not middle...

Reply:
Okay yup, then I think I would go through and remove all mention of b3 and replace it with "right-click" (sort of mentioned in another comment below)